### PR TITLE
feat(Radio): design + remove 'core' dependency

### PR DESCRIPTION
**Pull request by jmevel**

### CODE REVIEW

**Review by buddyvegas — DISMISSED** [review]

File: packages/css/src/Form/Radio/Client/Radio.scss [hunk 1]
```diff
@@ -80,91 +86,98 @@
...
   &__radio-inline,
   &__radio-custom {
     position: relative;
+    display: flex;
     width: 100%;
     border-radius: 4px;
     align-items: center;
-    box-shadow: 0 0 0 1px common.$color-gray inset;
-    cursor: pointer;
+    background-color: white;
```
Comment (GuillaumeKESTEMAN) [inline packages/css/src/Form/Radio/Client/Radio.scss]: use common color ?

File: packages/react/src/Form/Radio/Client/RadioItem.tsx [hunk 2]
```diff
@@ -1,60 +1,87 @@
...
       icon,
-      choice,
+      title,
+      subtitle,
       description,
-      classModifier = "",
-      className = "",
-      disabled = false,
+      className,
+      checked,
```
Comment (GuillaumeKESTEMAN) [inline packages/react/src/Form/Radio/Client/RadioItem.tsx]: `checked` can be in `...otherProps` no ?

File: packages/css/src/Form/Radio/Client/Radio.scss (hunk 1 above)
Comment (jmevel) [reply packages/css/src/Form/Radio/Client/Radio.scss]: Fixed

File: packages/react/src/Form/Radio/Client/RadioItem.tsx (hunk 2 above)
Comment (jmevel) [reply packages/react/src/Form/Radio/Client/RadioItem.tsx]: That's correct. 
Fixed !

**Review by GuillaumeKESTEMAN — APPROVED** [review]

**Review by buddyvegas — APPROVED** [review]

**Pull request merged**